detectors/node/opentelemetry-resource-detector-aws/test/detectors/AwsEc2Detector.test.ts

```diff
@@ -109,7 +109,8 @@ describe('awsEc2Detector', () => {
       scope.done();
     });
 
-    it('should throw when timed out', async () => {
+    it('should throw when timed out', function (done) {
+      this.timeout(7000);
```

Comment:
if you're changing the timeout value for the test, you're not really testing the new value you changed to. Don't you want to test with the 5s?

Reply:
Updated the timeouts